src/eduid_webapp/phone/views.py

```diff
@@ -187,48 +195,29 @@ def verify(user, code, number):
             '_status': 'error',
             'message': 'phones.code_expired_send_new'
         }
-
     if code != state.verification.verification_code:
         msg = "Invalid verification code: {!r}".format(state.verification)
         current_app.logger.debug(msg)
         return {
             '_status': 'error',
             'message': 'phones.code_invalid'
         }
-
-    current_app.proofing_statedb.remove_state(state)
```

Comment:
I don't see phone state removed anywhere now?

Reply:
It is done in the `verify_phone_number` function, in the `eduid_webapp.phones.verifications` module.